### ISSUE TITLE
Strange behavior for refrigerant plot (log(p)-h-Diagram)

### ISSUE THREAD

**Issue by yodifone** [CoolProp/CoolProp#1805]
Hi Ian or others.
There is a strange behavior of the plotting function. When I try to plot a log(p)-h-Diagram from Propane everything worked fine within an environment based on python 3.7.2  with an installed CoolProp 6.1.1dev. See pic1.png. ![pic1](https://user-images.githubusercontent.com/5373148/53998699-f378bf80-4140-11e9-98a4-039afa9bde75.png)
When I switch over to CoolProp 6.1.0 which is installed normally when doing "pip install coolprop" or  to  CoolProp 6.2.999 installed with this dreamhoster.com path also for pip the plot is wrong. See pic2.png. 
![pic2](https://user-images.githubusercontent.com/5373148/53998619-b7455f00-4140-11e9-9207-e95caef2cfab.png) This can be reproduced when doing the same installation procedure for an Python 2.7.1 and Python 3.4.2.
Can anyone help me to narrow down this presumably property-driven bug? 
Thanks inadvance!
Kind Regards.

**Comment by ibell** [comment]
Weird, why are you not getting 6.2.1 when you pip install?  

The plotting routines are not being actively maintained or developed sadly.  Can you contribute some debugging?

**Comment by yodifone** [comment]
Hi all.
I could upload stdout and stderr with debug level set to 100. But these files are quite huge. I generated to of these outputs for pic1 (Python 3.7.2 with CoolProp 6.1.1dev on an Arch Linux system) and pic2 (Python 3.7.2 with CoolProp 6.2.999 on Debian Jessie but within a recent (yesterday) miniconda3 environment for Python 3.7.2). Each log file has 58MB. The diff is almost exactly double sized. If you direct me to some hot spots I should look for within these log files I can do that and extract the right things to post it here.
Kind Regards!

**Comment by ibell** [comment]
I'm afraid that won't help much, as you'll need to do the debugging
yourself, or to find someone to help you do the debugging.  But none of the
primary developers can help out right now.

**Comment by yodifone** [comment]
Okay, than I will go for it. I just meant a hint like "look for the critical temperature" or something related to it. I didn't want to put the debugging on someone else's shoulders.

**Comment by ibell** [comment]
Yeah, sorry, I'm not sure what to recommend in terms of debugging, it's
been a while since I had my fingers into the plotting routines.

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
